recipes/nbgrader/run_test.sh

```diff
@@ -0,0 +1,6 @@
+conda install --yes nodejs
```

Comment:
Nodejs is in the test/require secrion. This is an issue #2117 with nodejs not getting linked in consecutive conda build and conda test envs in staged recipe. So this is a hack until it becomed a feedstock (where builds work without this hack) or someone can give me a better fix :p